src/ui/units/wizard/components/Dialogs/DialogShapes/DialogShapes.tsx

```diff
@@ -96,11 +99,12 @@ const DialogShapes: React.FC<Props> = ({
     }, [onClose]);
 
     const onApplyButtonClick = React.useCallback(() => {
-        const shapesConfig: ShapesConfig = {
+        onApply({
             mountedShapes: shapesState.mountedShapes,
             fieldGuid: item.guid,
-        };
-        onApply(shapesConfig);
+            paletteType: shapesState.paletteType,
```

Comment:
The type of palette can be determined by the type of visualization. This is a calculated field, it does not need to be stored in the config